events/nats.go

```diff
@@ -0,0 +1,76 @@
+package events
+
+import (
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill-nats/v2/pkg/nats"
+	"github.com/ThreeDotsLabs/watermill/message"
+	nc "github.com/nats-io/nats.go"
+)
+
+var natsMarshaler = &nats.JSONMarshaler{}
+
+func newNATSPublisher(cfg PublisherConfig) (message.Publisher, error) {
+	logger := watermill.NewStdLogger(false, false)
```

Comment:
This is a blocking logger, but they will take an `io.Writer`. We should wrap zap, drop in zero, or soemthing.